compile_halomod_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+gcc -Wall -Wpedantic -g -O0 -I/usr/local/include -std=gnu99 -fPIC tests/ccl_test_halomod.c -o tests/ccl_test_halomod -L/usr/local/lib -lgsl -lgslcblas -lm -Lclass -lclass -lccl
```

Comment:
This whole file shouldn't be in the repo in the end

Reply:
Done